isb_web/main.py

```diff
@@ -10,23 +10,25 @@
 import sqlalchemy.orm
 import accept_types
 import pydantic
-from isamples_metadata.SmithsonianTransformer import SmithsonianTransformer
 
 from isb_web import database
 from isb_web import schemas
 from isb_web import crud
 from isb_web import config
 from isb_web import isb_format
+from isb_web import isb_solr_query
 from isamples_metadata.SESARTransformer import SESARTransformer
 from isamples_metadata.GEOMETransformer import GEOMETransformer
 from isamples_metadata.OpenContextTransformer import OpenContextTransformer
+from isamples_metadata.SmithsonianTransformer import SmithsonianTransformer
```

Comment:
This was just a weird import cleanup, I thought I had somehow forgotten to include the Smithsonian records in the transformers.